s3backgrounddelete/ut/cortx_s3_config_tests.py

```diff
@@ -24,9 +24,15 @@
 import pytest
 
 from s3backgrounddelete.cortx_s3_config import CORTXS3Config
+from s3confstore.cortx_s3_confstore import S3CortxConfStore
 
 CONFIG_LOG_DIR = "/var/log/seagate/s3/s3backgrounddelete"
+conf_home_dir = os.path.join(
+            '/', 'opt', 'seagate', 'cortx', 's3', 's3backgrounddelete')
+_conf_file = os.path.join(conf_home_dir, 'config.yaml')
+conf_url = 'yaml://' + _conf_file
 
+s3confstore = S3CortxConfStore(conf_url, "dummy")
```

Comment:
Constructor of CORTXS3Config() will call this.
Is there a need to call S3CortxConfStore externally?

Reply:
fixed. Please check. @AtitaShirwaikar